flexmeasures/ui/templates/base.html

```diff
@@ -85,20 +85,23 @@
                             <span class="caret"></span></a>
                         <ul class="dropdown-menu">
                             <li {% if current_user.has_role('anonymous') %}class="disabled" {% endif %}><a
-                                    {% if not current_user.has_role('anonymous') %}href="/tasks/forecasting"
+                                    {% if not current_user.has_role('anonymous') %}href="/tasks/0/view/jobs/forecasting/started/10/1"
```

Comment:
What dies the `10` mean, here?

What is this URL path telling me in general?

Reply:
- 10/1 is a pagination instruction: 10 per page, and load page 1
- I opted to show ongoing jobs. Two alternatives I considered: showing failed jobs, or showing pending jobs. Each has its merits.